x-pack/plugin/ml/qa/ml-with-security/build.gradle

```diff
@@ -72,6 +72,8 @@ integTest.runner  {
     'ml/data_frame_analytics_crud/Test get stats given expression without matches and allow_no_match is false',
     'ml/data_frame_analytics_crud/Test delete given missing config',
     'ml/data_frame_analytics_crud/Test max model memory limit',
+    'ml/data_frame_analytics_memory_usage_estimation/Test memory usage estimation for empty data frame',
+    'ml/data_frame_analytics_memory_usage_estimation/Test memory usage estimation for non-empty data frame',
```

Comment:
Why do these have to be blacklisted from the security tests?  Usually the tests that need blacklisting are tests that return exceptions prior to checking security.  But these ones sound like they shouldn't be like that.

Reply:
I added them here because they were failing on CI.
TBH I didn't investigate the issue very thoroughly. If you take a look at this file, most (if not all) of our df analytics yml tests are here.

Reply:
FYI: I reverted this change and will investigate once CI finishes.

Reply:
FYI: Missing privilege added in a separate PR: https://github.com/elastic/elasticsearch/pull/45451